app/views/katello/api/v2/repositories/show.json.rabl

```diff
@@ -6,6 +6,9 @@ extends 'katello/api/v2/common/timestamps'
 attributes :content_type
 attributes :docker_upstream_name
 attributes :unprotected, :full_path, :checksum_type, :container_repository_name
+if (@object && @object.yum?) || (@resource.is_a?(Katello::Repository) && @resource.yum?)
```

Comment:
I would prefer we just show download policy on all repos. @ehelms your thoughts?